packages/import-bundle/test/do-import.js

```diff
@@ -0,0 +1,49 @@
+import bundleSource from '@agoric/bundle-source';
+import tap from 'tap';
+import { importBundle } from '../src/index.js';
+
+// function transform(rewriterState) {
```

Comment:
Note https://github.com/Agoric/SES-shim/pull/255

Reply:
oops, yep, I'll rewrite the test to match the actual API.